### ISSUE TITLE
Update heading level of Pagination section in the CMS

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#10111]
## Description
Within the CMS, there are visually hidden headings for various page sections.  The pagination at the bottom of the content listing page currently has an H4 which skips the H3 level in the heading structure for the page as a whole.

## Screenshot
![image](https://user-images.githubusercontent.com/106678594/183667064-bbbae827-1915-4715-bcaa-88b106576800.png)

## Accessibility Standard

WCAG version 2.0 A, [Criterion 1.3.1](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)

### Notes
Before we started this issue, I already found out that this was a core issues, therefore the original ACs were no longer relevant. For reference I have noted the original ACs here in case they were needed:

- Determine if this is Drupal Core or specific to VA
- Implement patch if it exists
- If no patch exists, confirm that changing the heading to an H3 would be appropriate for all instances or if this needs to have a different heading level in different instances
- Update Heading level
- Accessibility lead review

## Acceptance Criteria

- [x] Determine if we should push core issues forward or update ourselves
- [ ] Accessibility lead review when issue is updated

**Comment by laflannery** [comment]
This is a core issue that is currently being worked on: https://www.drupal.org/project/drupal/issues/3333401 So we can probably punt this and just wait this it's updated in core?

**Comment by productmike** [comment]
@edmund-dunn @laflannery 
I see the last comment on this from Laura is that this may be addressed in a core patch. If that's the case, wondering if this would be package in D10 and we continue to wait? Or if not, do we have a sense on the timing from the core patch and if better to go ahead an do this instead of waiting longer?

**Comment by laflannery** [comment]
@productmike From what I can tell this hasn't been updated yet, the last comment on the issues was 3 months ago. Edmund would have a better idea of how things like this work - if we can push it forward or whatnot

**Comment by productmike** [comment]
re: https://dsva.slack.com/archives/C052A7RJFPS/p1685638469339529, pulling this out of s85

**Comment by tonytaylor** [comment]
We help push core along to a solution on this one.
Please review the following [document](https://docs.google.com/document/d/1lUSa5MkEz0LPjWIvJoUt12IHuwT6CTEqtBa9PGTYUco/edit?usp=sharing)

**Comment by tonytaylor** [comment]
Finishing up testing of static patch rendered from the MR. This should be up on d.o by EOD.

**Comment by tonytaylor** [comment]
Continuing work on this. Posted to d.o and currently failing a test on 10.1.x.  Currently investigating and suspect a side-effect (nodes created not properly appearing in view) is at play.

**Comment by BerniXiongA6** [comment]
Hey @tonytaylor -- how close are we to completing this ticket?

**Comment by tonytaylor** [comment]
@BerniXiongA6 -- still working through a schema violation issue within the patch.  I think I may confer with @edmund-dunn with regard to the deets, but I believe we're close on this one.

**Comment by laflannery** [comment]
Screenshot of HeadingsMap plugin showing error
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/a914a1e3-4db7-455c-b46d-eeb5b5890019)

Screenshot of Tugboat showing new heading structure without error:
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/4bd99e4d-2d6a-4a54-9e6d-9660b3abad5e)

**Comment by productmike** [comment]
@tonytaylor We left this at pushing it forward and making a comment with d.o to mention that one of their test were failing, as well as work with @ndouglas on a cypress test case to ensure this continually passes our testing (though believe that was moreso for future state). Can we consider these to be completed and move this ticket forward?
